.github/workflows/run-automated-test.yml

```diff
@@ -0,0 +1,28 @@
+name: Run automated tests on push to master
+
+on:
+  push:
```

Comment:
You should be able to remove the "branches" designation all together so it will run on any branch that gets pushed.  This will allow you to test your Actions without waiting for it to be merged into master.